images/win/scripts/Installers/Install-SeleniumWebDrivers.ps1

```diff
@@ -44,16 +45,39 @@ Write-Host "Chrome driver install....";
 Expand-Archive -Path "$ChromeDriverPath\chromedriver_win32.zip" -DestinationPath $ChromeDriverPath -Force;
 Remove-Item -Path "$ChromeDriverPath\chromedriver_win32.zip" -Force;
 
-Write-Host "Setting the environment variables";
+# Install Microsoft Edge Web Driver
+Write-Host "Microsoft Edge driver download...."
+$EdgeDriverPath = "$DestinationPath\SeleniumWebDrivers\EdgeDriver"
+if (-not (Test-Path -Path $EdgeDriverPath)) {
+    New-Item -Path $EdgeDriverPath -ItemType "directory"
+}
+
+$EdgePath = (Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\msedge.exe').'(default)'
```

Comment:
Path looks like a hardcode, is it possible to workaround that or we had to hardcode?

Reply:
Done, thanks